src/device.rs

```diff
@@ -112,12 +112,11 @@ impl<'a> Device<'a> {
 
     /// Returns the subsystem name of the device.
     ///
+    /// Returns None if the device does not belong to a subsystem.
```

Comment:
I think this is implied by the function signature. The other functions that return `Option<T>` don't document the `None` value.